### PR TITLE
feat(responsive): Make home page responsive

### PR DESCRIPTION
**Pull request by prashanthr**

### CODE REVIEW

File: src/components/blockchain/index.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ export default class Blockchain extends Component {
             <br />
             <div>
               Cryptocurrencies leverage blockchain to bring financial freedom and stability to more people
-              in today's global economy.
+              in today&#8217;s global economy.
```
Comment (DSofter) [inline src/components/blockchain/index.js]: Maybe use `’` directly?

File: src/components/subscribe-form/index.css [hunk 2]
```diff
@@ -44,24 +47,46 @@
...
 .subscribe-form-input {
-  padding: 10px 16px;
+  padding: 10px;
+  /* min-height: 50px; instead of padding*/
   font-size: 18px;
   line-height: 1.3333333;
-  width: 300px;
   color: black;
+  display: block;
+  /* width: 300px; */
```
Comment (DSofter) [inline src/components/subscribe-form/index.css]: Maybe remove the comments?

**Comment by DSofter** [comment]
Awesome work thanks!

File: src/components/blockchain/index.js (hunk 1 above)
Comment (prashanthr) [reply src/components/blockchain/index.js]: You are absolutely correct. I was thinking about older browser / backward compatibility but We have specified UTF-8 so it should be okay.

File: src/components/subscribe-form/index.css (hunk 2 above)
Comment (prashanthr) [reply src/components/subscribe-form/index.css]: Yes thanks! It was going to be part of my cleanup before merging.

**Pull request merged**